### ISSUE TITLE
Spark-CSV is not able to load ZIP files

### ISSUE THREAD

**Issue by umeshkacha** [databricks/spark-csv#440]
Hi spark-csv seems to be not supporting loading zip files. I tried codec bzip2 but it does not work. If I convert zip file into gz file then I can load same file without any problem. Please guide.

**Comment by HyukjinKwon** [comment]
I don't think it is this library's specific issue. Were you able to read it in Spark's other text based datasources such as JSON or TEXT?
